uchiwa/daemon/daemon.go

```diff
@@ -42,7 +42,7 @@ type DatacenterSnapshot struct {
 	Checks     []interface{}
 	Clients    []interface{}
 	Events     []interface{}
-	Info       structs.Info
+	Info       *structs.Info
```

Comment:
Why make this a pointer? Not saying it's wrong to have a pointer here, just curious about the reasoning.

Reply:
I made it a pointer so that it can be assigned & checked for a nil value.